assets/js/theme/global/modal.js

```diff
@@ -230,6 +234,10 @@ export class Modal {
     }
 
     onModalOpened() {
+        if (this.pending) {
+            this.$modal.one(ModalEvents.loaded, () => this.setupFocusTrap());
```

Comment:
important: Do we expect one here?

Reply:
Yes. Event listener should be removed after execution